include/rofl/ofdpa/rofl_ofdpa_fm_driver.hpp

```diff
@@ -40,6 +40,8 @@ namespace rofl {
 namespace openflow {
 
 class rofl_ofdpa_fm_driver final {
+  static const uint16_t DEFAULT_MAX_LEN = 1522;
```

Comment:
my recommendation would be to use a higher value for this that either is matching the current value of of-dpa or `OFPCML_NO_BUFFER`